### DISCUSSION TITLE
Bug: Import field with zeros at beginning, text field number imported as scientific notation

### DISCUSSION THREAD

**Discussion by stellarfinn** [Budibase/budibase#11523] [Help]
When importing data, any fields that start with zeros will import incorrectly. The zeros will be dropped and the row will not update, it will create a new row. See screenshot...

I downloaded an export of row 1. Updated the CiaoSalesChannels to be only "Shopify" and then imported to update the row. Instead of updating row 1 with MfgCode "00018"... it created a new row 14 with MfgCode "18" (removing the preceding three zeros).

Also, it imported the CaseUPC field as scientific notation instead of a string. 

<img width="1581" alt="image" src="https://github.com/Budibase/budibase/assets/99847728/705bbbd4-6e16-4152-9076-b1c652aa5cfb">

**Comment by ConorWebb96** [comment]
Hey @stellarfinn,

I wasn't able to replicate this behaviour with the text field.

What version of Budibase are you running?
Are you self-hosted or cloud?
What data source are you using? I tested this on MySQL and the Budibase internal table.

I can see it is a text field. Did you edit the CSV manually before re-importing it back into the table? Where there any additional steps taken before re-import?

Thanks in advance for the information!
